module/VuFindSearch/src/VuFindSearch/Backend/EDS/Connector.php

```diff
@@ -136,20 +136,4 @@ protected function httpRequest(
         }
         return $resultBody;
     }
-
-    /**
-     * Print a message if debug is enabled.
-     *
-     * @param string $msg Message to print
-     *
-     * @return void
-     */
-    protected function debugPrint($msg)
```

Comment:
This one can be removed without deprecation since the parent class has another version.